atoma-service/src/handlers/chat_completions.rs

```diff
@@ -244,15 +244,15 @@ pub async fn chat_completions_handler(
         is_stream,
         payload.clone(),
         num_input_tokens,
-        estimated_total_compute_units,
+        estimated_output_compute_units,
         client_encryption_metadata,
         headers,
     )
     .await
     {
         Ok(response) => {
             CHAT_COMPLETIONS_ESTIMATED_TOTAL_TOKENS.add(
-                estimated_total_compute_units,
+                num_input_tokens + estimated_output_compute_units,
```

Comment:
Shouldn't this be `estimated_inpt_compute_units + estimated_output_compute_units` ?